test/Test/Oscoin/DummyLedger.hs

```diff
@@ -62,6 +63,10 @@ type DummyState = [DummyTx]
 
 type DummyOutput = DummyTx
 
+-- -- | Prepends the transaction to the state and output the transaction.
+-- dummyEval :: Evaluator DummyState DummyTx DummyOutput
+-- dummyEval tx txs = Right (tx, tx:txs)
```

Comment:
Can we kill this?

Reply:
Yes indeed! Friday afternoon artifact ;)